metricflow-semantics/metricflow_semantics/naming/linkable_spec_name.py

```diff
@@ -29,6 +27,18 @@ class StructuredLinkableSpecName:
     time_granularity_name: Optional[str] = None
     date_part: Optional[DatePart] = None
```

Comment:
Let's remove these and define property accessor methods instead.

Reply:
Oops that was just an oversight 🤦‍♀️ We shouldn't need to add property methods, having public attributes in the init should cover it.